tryy/cna.py

```diff
@@ -0,0 +1,100 @@
+# -!- coding: utf-8 -!-
+
+from tryy.base_class import Base
+
+
+def cna_crawler(size=30):
+
+    media = '中央社'
+    headers = {
+        "User-Agent":
+            "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/119.0.0.0 Safari/537.36"
+    }
```

Comment:
This should be the member of base class because all crawlers use the same header.